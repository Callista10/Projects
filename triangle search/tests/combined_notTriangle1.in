-N=1 -mindist=3 -rseed=5