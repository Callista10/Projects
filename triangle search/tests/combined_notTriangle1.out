read 1 points
12.29, 24.18
This is not a triangle
